.travis.yml

```diff
@@ -1,10 +1,15 @@
 language: objective-c
 osx_image: xcode7.1
 
+before_install:
+  - brew update
+  - brew install swiftlint 
+
 install:
-    - gem install xcpretty
-    - gem install cocoapods
-    - pod install
+  - gem install xcpretty
+  - gem install cocoapods
+  - pod install
```

Comment:
gem install xcpretty
gem install cocoapods

Should be moved in the before_install and only keep the pod install